src/test/java/com/mongodb/hibernate/internal/translate/mongoast/command/aggregate/AstMatchStageTests.java

```diff
@@ -35,7 +35,8 @@ void testRendering() {
                 new AstComparisonFilterOperation(EQ, new AstLiteralValue(new BsonString("Jane Eyre"))));
         var astMatchStage = new AstMatchStage(astFilter);
 
-        var expectedJson = """
+        var expectedJson =
+                """
                 {"$match": {"title": {"$eq": "Jane Eyre"}}}\
                 """;
         assertRender(expectedJson, astMatchStage);
```

Comment:
the above issue was rectified simply due to the fact that our format steps happen AFTER `palantir`